### PR TITLE
[JBEAP-14477] workaround for no physical address for foo, dropping message warnings

### PR DESCRIPTION
**Pull request by luck3y**
<groups transport: <property name="use_ip_addrs">true</property>

### CODE REVIEW

**Comment by luck3y** [comment]
Note @rcernich this is EAP-CD only. I'm not sure if linking a JBEAP issue is OK, or if we want a CLOUD issue too?

**Comment by luck3y** [comment]
I only saw this on CD in testing, it never happened on eap71.

File: jboss-eap-cd-openshift-launch/added/standalone-openshift.xml [hunk 1]
```diff
@@ -420,7 +420,9 @@
                     <protocol type="FRAG3"/>
                 </stack>
                 <stack name="tcp">
-                    <transport type="TCP" socket-binding="jgroups-tcp"/>
+                    <transport type="TCP" socket-binding="jgroups-tcp">
+                       <property name="use_ip_addrs">true</property>
```
Comment (rhusar) [inline jboss-eap-cd-openshift-launch/added/standalone-openshift.xml]: Ken,  our profiles are using 4 spaces; not 3.

Comment (luck3y) [reply jboss-eap-cd-openshift-launch/added/standalone-openshift.xml]: Ah, well spotted :)

Comment (luck3y) [reply jboss-eap-cd-openshift-launch/added/standalone-openshift.xml]: WS change is here: https://github.com/jboss-container-images/jboss-eap-modules/pull/6/files

@rcernich Can you update https://issues.jboss.org/browse/CLOUD-2425 to be in Sprint 16? I'll link the PRs once it's in. It seems weird to have  JBEAP driving this.

**Pull request merged**